ca/certificate-authority.go

```diff
@@ -337,7 +365,20 @@ func (ca *CertificateAuthorityImpl) GenerateOCSP(xferObj core.OCSPSigningRequest
 		RevokedAt:   xferObj.RevokedAt,
 	}
 
-	ocspResponse, err := ca.ocspSigner.Sign(signRequest)
+	cn := cert.Issuer.CommonName
```

Comment:
We may want to add a comment here indicating that while it is possible to have multiple issuers with the same common name Boulder won't support that (since it isn't a good idea to do so generally), or if we want to I guess using a subject + authority key ID fingerprint would also be possible.